.editorconfig

```diff
@@ -188,7 +188,7 @@ dotnet_analyzer_diagnostic.category-Naming.severity           = warning
 dotnet_analyzer_diagnostic.category-Performance.severity      = warning
 dotnet_analyzer_diagnostic.category-Reliability.severity      = warning
 dotnet_analyzer_diagnostic.category-Security.severity         = error
-dotnet_analyzer_diagnostic.category-Style.severity            = error
+dotnet_analyzer_diagnostic.category-Style.severity            = warning
```

Comment:
Can we keep this as an error?

Reply:
This should be fine with Joe's change.  We still have warnings-as-errors for everything but the local VS IDE build, so setting it to warning here should speed up the inner loop while still being errors before check-in.